Classes/Routing/Router.swift

```diff
@@ -492,6 +492,16 @@ final class Router:
                     )
                 )
             }
+        case .qrScanner:
+            let session = self.appConfiguration.session
+            if let authenticatedUser = session.authenticatedUser,
```

Comment:
Use `guard` to avoid unnecessary nesting. 

Consider refactor this code to:
```swift
guard let authenticatedUser = appConfiguration.session.authenticatedUser, authenticatedUser.accounts.isNonEmpty else { return }
route(
    to: .qrScanner(canReadWCSession: true),
    from: findVisibleScreen(over: rootViewController),
    by: .present
)
```